go.mod

```
@@ -21,17 +21,17 @@ require (
 	github.com/containernetworking/plugins v1.3.0
 	github.com/containers/common v0.55.4
 	github.com/containers/conmon v2.0.20+incompatible
-	github.com/containers/conmon-rs v0.5.1
+	github.com/containers/conmon-rs v0.6.1
 	github.com/containers/image/v5 v5.27.0
 	github.com/containers/kubensmnt v1.2.0
 	github.com/containers/ocicrypt v1.1.8
 	github.com/containers/podman/v4 v4.6.2
-	github.com/containers/storage v1.49.0
+	github.com/containers/storage v1.50.0
```

```suggestion
	github.com/containers/storage v1.50.2
```

I wanted to bump this up in https://github.com/cri-o/cri-o/pull/7313 but you already had this PR up.

---

Let's follow-up on this, the PR almost got it through CI already.